beetsplug/vocadb.py

```diff
@@ -1,12 +1,18 @@
+from collections.abc import Iterable, Sequence
 from datetime import datetime
 from itertools import chain
 from json import load
+from optparse import Values
 from re import Match, match, search
-from typing import Any, NamedTuple, Optional, Sequence
+from typing import NamedTuple, TypedDict, TYPE_CHECKING
+from typing_extensions import NotRequired, override
```

Comment:
Since these exist in newer versions of python, we should import them from `typing` if possible.

```suggestion
from sys import version_info

if sys.version_info >= (3, 11):
    from typing import NotRequired
else:
    from typing_extensions import NotRequired
if sys.version_info >= (3, 12):
    from typing import override
else:
    from typing_extensions import override
```